packages/rum-core/src/domain/rumEventsCollection/error/errorCollection.ts

```diff
@@ -26,18 +26,26 @@ export function startErrorCollection(lifeCycle: LifeCycle, configuration: Config
 }
 
 export function doStartErrorCollection(lifeCycle: LifeCycle, observable: Observable<RawError>) {
-  observable.subscribe((error) => lifeCycle.notify(LifeCycleEventType.RAW_RUM_EVENT_COLLECTED, processError(error)))
+  observable.subscribe((error) => lifeCycle.notify(LifeCycleEventType.RAW_ERROR_COLLECTED, { error }))
```

Comment:
nit: by wiring the observable to the raw error event, we could now: 
- remove the observable from doStartErrorCollection signature
- wire the automaticErrorCollection to lifecycle in parent function